### DISCUSSION TITLE
2.9.0

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#522] [Releases]
# Upscayl v2.9.0 🥳
<img width="475" alt="image" src="https://github.com/upscayl/upscayl/assets/25067102/0a8379bf-471a-4796-9692-a8203e8262b8">

> [!NOTE]
**PRE-RELEASE ONLY FOR OUR LINUX GANG 🐧❤️**
Please report bugs if you notice any 🙏

## What's Changed 🤓
* We're launching Upscayl on the Apple App Store soon 🥳
* **NCNN update!** Vulkan SDK and a lot of libraries have been updated, expect bug fixes and performance improvements💯 
* Fix #499 - Remove redundant upscaled files 🗑️
* Fix #488 - Batch Upscale empty progress bug 🪹
* Security updates 👮‍♀️
* Skip conversion for 4x PNG upscaling ⏭️

**Full Changelog**: https://github.com/upscayl/upscayl/compare/v2.8.6...v2.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/upscayl/upscayl/releases/tag/v2.9.0'>2.9.0</a>.</em>

**Comment by aaronliu0130** [comment]
Just the Mac App Store, right?

**Reply by NayamAmarshe** [reply]
Yes
